Duplicati/Library/Backend/Mega/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="MegaApiClient" version="1.7.1" targetFramework="net471" />
+  <package id="MegaApiClient" version="1.9.0" targetFramework="net471" />
   <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net471" />
-</packages>
+  <package id="Otp.NET" version="1.2.2" targetFramework="net471" />
```

Comment:
Can you add the corresponding license information for Otp.NET to the `Duplicati.License` project?

Reply:
Done!